site/content/docs/commands/run-local.en.md

```diff
@@ -8,14 +8,17 @@ $ copilot run local [flags]
 
 ## What are the flags?
 ```
-  -a, --app string                        Name of the application. (default "playground")
```

Comment:
😂 😂

Reply:
I didn't even mean to remove this cause I just copied the results from `copilot run local -h` but I don't know why this was here in the first place lol